Resources/Prototypes/_Nuclear14/Entities/Objects/Misc/junk.yml

```diff
@@ -225,6 +238,12 @@
   components:
   - type: Sprite
     state: crutch
+ - type: WelderRefinable
+    refineResult:
+    - MaterialWoodPlank2
+    - N14JunkComponentScrew1
```

Comment:
`N14JunkComponentScrew` is not a stacking material

Reply:
thanks for all these comments, will read on that link for sure, ill try changing them later, thanks again.